src/service/viewer-impl.js

```diff
@@ -608,6 +608,29 @@ export class Viewer {
   }
 
   /**
+   * Triggers "tick" event for the viewer.
+   * @param {!JSONObject} message
+   */
+  tick(message) {
+    this.sendMessage_('tick', message, false);
+  }
+
+  /**
+   * Triggers "sendCsi" event for the viewer.
+   */
+  sendCsi() {
+    this.sendMessage_('sendCsi', undefined, false);
+  }
+
+  /**
+   * Triggers "setParams" event for the viewer.
+   * @param {!JSONObject} message
+   */
+  setParams(message) {
+    this.sendMessage_('setParams', message, false);
```

Comment:
Could we use a different name? "setParams" seems a bit vague to me. Maybe "setFlushParams" or "setInstrumentationParams"